eng/common/scripts/Start-DevOpsBuild.ps1

```diff
@@ -0,0 +1,55 @@
+<#
+.SYNOPSIS
+Starts an Azure DevOps pipeline with the requested variables
+
+.PARAMETER DevOpsPat
+DevOps PAT obtained from https://docs.microsoft.com/en-us/azure/devops/organizations/accounts/use-personal-access-tokens-to-authenticate?view=azure-devops&tabs=preview-page
+
+.PARAMETER DevOpsOrg
+DevOps organization name (used to build API URL)
+
+.PARAMETER DevOpsProject
+DevOps project name (used to build API URL)
+
+.PARAMETER DevOpsPipelineId
+ID of the pipeline to launch (used to build API URL)
+
+.PARAMETER RequestBodyJson
+Body JSON of the request. Example: 
+'{ "variables": { "params": { "value": "{ \"target_repo\": { \"url\": \"https://github.com/MicrosoftDocs/azure-docs-sdk-node\", \"branch\": \"<target-branch-name>\", \"folder\": \"./\" }, \"source_repos\": [] }" } } }'
+#>
+
+param(
+  [Parameter(Mandatory = $true)]
+  [string] $DevOpsPat,
+
+  [Parameter(Mandatory = $true)]
+  [string] $DevOpsOrg,
+
+  [Parameter(Mandatory = $true)] 
+  [string] $DevOpsProject,
+
+  [Parameter(Mandatory = $true)]
+  [string] $DevOpsPipelineId,
+
+  [Parameter()]
+  [string] $RequestBodyJson
+)
+
+# Create a basic auth header part with a prepended colon to 
+# represent that no username is set.
+# https://docs.microsoft.com/en-us/azure/devops/organizations/accounts/use-personal-access-tokens-to-authenticate?view=azure-devops&tabs=preview-page#use-a-pat-in-your-code
+$authHeader = [Convert]::ToBase64String([System.Text.Encoding]::UTF8.GetBytes(":$DevOpsPat"))
+
+$result = Invoke-RestMethod `
```

Comment:
Should we be calling https://github.com/Azure/azure-sdk-for-js/blob/main/eng/common/scripts/Invoke-DevOpsAPI.ps1#L12.

Reply:
This could be possible with some modification of this script according to the `api-version=6.0` https://docs.microsoft.com/en-us/rest/api/azure/devops/build/builds/queue?view=azure-devops-rest-6.0